client/my-sites/post-type-list/post-actions-ellipsis-menu/qrcode.jsx

```diff
@@ -0,0 +1,73 @@
+import { Icon } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+import { getPost } from 'calypso/state/posts/selectors';
+import PostActionsQRCode from '../../post-actions-qr-code';
+
+class PostActionsEllipsisMenuQRCode extends Component {
+	static propTypes = {
+		globalId: PropTypes.string,
```

Comment:
Seems like this component won't do much without a `globalId`. Should this be a required prop?

Reply:
It seems that specifying it as a required prop is throwing a console error. Probably because it is trying to render all menu options on loading placeholder and there's no `globalId` at that moment. I will leave it as optional for now (as we do currently on the others menu items) and will revisit this in another PR.